doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1761,23 +1829,23 @@ OCX-189: When inputting -40 dBm to HCI_OPCODE_VS_SET_CONN_TX_PWR (0x3F6), the ac
   The TX power for a connection cannot be less than -20 dBm.
   Controller still output -20 dBm if the setting is -40 dBm.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCX-208: There is a chance that the scan report only shows legacy-ADV without ADV-EXT
   This might happen if a broadcast sink is moved in and out of radio range from the broadcast source or if the broadcast source is being rapidly reset.
   It will cause a broadcast sink to not be able to finish PA sync.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCX-217: The controller can get unresponsive when a broadcast sink tries to sync to some broadcast sources
   This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCX-218: The BIG info is not sent from the controller when a broadcast sink tries to sync to some broadcast sources
   This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCX-223: The controller asserts on the CIS gateway if two paused headsets reset
   If a CIS headset pair is reset simultaneously a few times while remaining paused, the controller on the gateway might get unresponsive.
```

Comment:
Please add:
OCX-NONE: OCT-2897:
Use of interleaved packing on the controller may cause left headset to disconnect.

Reply:
Only for v2-6-0?

Reply:
Yes